TheFourFifths/consus-client
# display fault history on item tff218

### Summary

Added ability to see fault history on item page

### Checklist


### How to Review

1. Add fault to item.
2. clear fault
2. See it in history

### Links

- [TFF-218](https://msoese.atlassian.net/browse/TFF-218)
